inloop/testrunner/runner.py

```diff
@@ -160,8 +161,14 @@ def communicate(self, task_name, input_path, output_path):
             LOG.debug("removing timed out container %s", ctr_id)
             subprocess.call(["docker", "rm", "--force", str(ctr_id)], stdout=subprocess.DEVNULL)
 
-        stderr = stderr.decode("utf-8", errors="replace")
-        stdout = stdout.decode("utf-8", errors="replace")
+        if stderr:
```

Comment:
Out of curiosity: does piping to `stdout` mean `stderr` will be `None`? Or why is this check necessary? `"".decode()` should work afaik.